receivers/discord/discord_test.go

```diff
@@ -555,18 +565,25 @@ func TestNotify_WithImages(t *testing.T) {
 			UseDiscordUsername: false,
 		}
 
+		tokenStore := images.NewFakeTokenStore(15)
+		imageProvider := images.NewTokenProvider(tokenStore, &logging.FakeLogger{})
+
 		// Create 10 alerts with an image each, Discord's embed limit is 10, and we should be using a maximum of 9 for images.
```

Comment:
I don't think this comment is relevant, right?
```suggestion
```

Reply:
I meant that this comment is not accurate. We're creating 15 alerts in this test, not 10.